db-service/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.18.0](https://github.com/cap-js/cds-dbs/compare/db-service-v1.17.2...db-service-v1.18.0) (2025-02-13)
+
+
+### Added
+
+* support for `SELECT.hint` ([#1004](https://github.com/cap-js/cds-dbs/issues/1004)) ([2c6a763](https://github.com/cap-js/cds-dbs/commit/2c6a7634450047cab22322abb7312f8f3d432cdf))
```

Comment:
remove before release, this is a hana only feat and has been accidentally introduced through: https://github.com/cap-js/cds-dbs/pull/1004/files#diff-d9ef4ae327f7e5954027eae8acf2866e34a6a365a11e60a3dc5ff535a210c33fL267